app/views/list/remove-modal.html

```diff
@@ -0,0 +1,24 @@
+<div class="modal-header">
+	<h4 class="modal-title">
+		Delete list
+	</h4>
+</div>
+
+<div class="modal-body">
+
+	Are you sure you want to delete list <strong>{{list.name}}</strong>?
+
+</div>
+
+<div class="modal-footer">
+
+	<button class="btn btn-danger" type="button" ng-click="remove()">
+		<i class="fa fa-exclamation-triangle"></i>
```

Comment:
Why did you use exclamation instead of trash?? 0_o Now there is two buttons with name Delete where one have trash icon and another one have exclamation... 
And by the way there is question: "Are you sure you want to delete list {{}}?" so i think the button here shouldn't be delete and cancel but Yes and No